### ISSUE TITLE
How to make language required? (do not allow language = "none")

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#673]
I see no such setting on the content type config page. Am I missing something?

**Comment by quicksketch** [comment]
I don't think there is such an option. Did one exist in Drupal 7 core?

**Comment by klonos** [comment]
Yeah, you are right, D7 core doesn't have such an option (nor does D8). You need i18n for that (the Multilingual Content submodule to be exact):

![i18n_d7-multilingual_settings-require_language](https://cloud.githubusercontent.com/assets/2423362/5991316/563670ec-aa3a-11e4-9454-d5bc8cb42297.png)

**Comment by klonos** [comment]
Is there any chance to add this feature in core? I mean, do we plan to let i18n be ported in contrib or are some of its features planned for core? Asking because I realize that it is a huge/complicated project.

**Comment by quicksketch** [comment]
> I mean, do we plan to let i18n be ported in contrib or are some of its features planned for core?

It's a mixed bag here for sure. i18n module provides a combination of critical functionality and sort-of edge-case functionality. This list of screenshot options is a great example. Which of those options is suitable for core and which are better for contrib? We'll need i18n module for Backdrop 1.x for sure, so I expect it will be ported while important portions of it are moved into core.
